nucypher/blockchain/eth/sol/source/contracts/Adjudicator.sol

```diff
@@ -138,14 +126,14 @@ contract Adjudicator is Upgradeable {
         // A task specification is: capsule + ursula pubkey + alice address + blockhash
         bytes32 stampXCoord;
         assembly {
```

Comment:
could you explain what the next few lines do and why it can't be achieved in vanilla solidity?